### ISSUE TITLE
fix fade mode bug

### ISSUE THREAD

**Issue by yyman001** [nolimits4web/Swiper#2065]
This is a fix fade mode bug.
* [fade mode] bug
* Swiper Version: all

### What you did
by `Effects`  in `2509` row
before:
```javascript
                for (var i = 0; i < s.slides.length; i++) {
                        var slide = s.slides.eq(i);
                        var offset = slide[0].swiperSlideOffset;
                        var tx = -offset;
                        if (!s.params.virtualTranslate) tx = tx - s.translate;
                        var ty = 0;
                        if (!s.isHorizontal()) {
                            ty = tx;
                            tx = 0;
                        }
                        var slideOpacity = s.params.fade.crossFade ?
                                Math.max(1 - Math.abs(slide[0].progress), 0) :
                                1 + Math.min(Math.max(slide[0].progress, -1), 0);
                        slide
                            .css({
                                opacity: slideOpacity
                            })
                            .transform('translate3d(' + tx + 'px, ' + ty + 'px, 0px)');
        
                    }
```
after
```javascript
                 var activeSlide = s.slides.eq(s.activeIndex)[0];
                    for (var i = 0; i < s.slides.length; i++) {
                        var slide = s.slides.eq(i);
                        var offset = slide[0].swiperSlideOffset;
                        var tx = -offset;
                        if (!s.params.virtualTranslate) tx = tx - s.translate;
                        var ty = 0;
                        if (!isH()) {
                            ty = tx;
                            tx = 0;
                        }

                      /*  var slideOpacity = s.params.fade.crossFade ?
                                Math.max(1 - Math.abs(slide[0].progress), 0) :
                                1 + Math.min(Math.max(slide[0].progress, -1), 0);*/

		                if (activeSlide !== slide[0]) {
			                slide.css({opacity: 0})//.transform('translate3d(' + tx + 'px, ' + ty + 'px, 0px)');
		                } else {
			                slide.css({opacity: 1})//.transform('translate3d(' + tx + 'px, ' + ty + 'px, 0px)');
		                }
		                slide.transform('translate3d(' + tx + 'px, ' + ty + 'px, 0px)');
                    }
```
### Actual Behavior
switch other slide will by opacity is 1 ,there will overlap other slide elemet . so , set no active slide opacity 
by 0, set active slide opacity by 1.
切换页面,其他slide有时候没有变为透明,跟其他的slide重叠了,将当前活动页设置为不透明,其他页面设置为透明.

**Comment by yyman001** [comment]
fix

**Comment by BulatSa** [comment]
Thanks! But in after
`if (!isH()) { `
fix to
`if (!s.isHorizontal()) {`

**Comment by Kisl0iD** [comment]
Thanks a lot! Very useful.

**Comment by smesko85** [comment]
@yyman001 Is this merged? I'm having described problem on 4.1.0

**Comment by yyman001** [comment]
@smesko85 Sorry，i dont't know how to merged for git pull .you can update this code for your code.
